docs/howto/manage-tracks.rst

```diff
@@ -16,7 +16,7 @@ new pattern. This document shows you how.
 Request a track guardrail
 -------------------------
 
-..    See also: :ref:`guardrail` (link to new Juju docs > charm > channel > track > guardrail)
+    See first: :external+juju:ref:`Juju | Charm channel track guardrail <guardrail>`
```

Comment:
There's no target in the Juju docs named `guardrail`, so we'll first need that added upstream.

Reply:
I suggest we merge this PR with as is and I will make sure to add the Juju anchors in the Juju PR. Or we give up on intersphinx. Otherwise we're forever stuck in a chicken and egg situation here.